iscdhcp/files/systemd_file.RedHat

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=DHCPv4 Server Daemon
+Documentation=man:dhcpd(8) man:dhcpd.conf(5)
+Wants=network-online.target
+After=network-online.target
+After=time-sync.target
+
+[Service]
+EnvironmentFile=-/etc/sysconfig/dhcpd
+Type=notify
+ExecStart=/usr/sbin/dhcpd -f -cf /etc/dhcp/dhcpd.conf -user dhcpd -group dhcpd --no-pid $DHCPDARGS
```

Comment:
this patch does more:
it brings back old behaviour of having /etc/sysconfig/dhcpd and set $DHCPDARGS for el7 hosts.
with this patch:
- on all rh's /etc/sysconfig/dhcpd is created with $DHCPDARGS for listen_interfaces
- a systemd file for dhcpd is created under /etc/systemd/system/dhcpd.service which overrides systemd unit file for dhcpd which comes with rpm package
  - /etc/systemd/system/dhcpd.service uses the "defaults file" and is therefor aware of $DHCPDARGS which are set in /etc/sysconfig/dhcpd